.gitlab-ci.yml

```diff
@@ -1960,7 +1960,7 @@ trigger_kitchen_functional_test:
     - .kitchen_azure_location_north_central_us
   <<: *kitchen_common
   rules:
-    - when: always
+    - when: manual
       allow_failure: true
   needs: [ "trigger_kitchen_functional_test", "tests_ebpf" ]
```

Comment:
Now that functional tests are manual, I believe we can remove the `trigger_kitchen_functional_test` job, it doesn't have any purpose anymore.

Reply:
Done